src/backend/distributed/commands/dependencies.c

```diff
@@ -207,7 +207,7 @@ ErrorIfCircularDependencyExists(const ObjectAddress *objectAddress)
 DeferredErrorMessage *
 DeferErrorIfCircularDependencyExists(const ObjectAddress *objectAddress)
 {
-	List *dependencies = GetAllSupportedDependenciesForObject(objectAddress);
+	List *dependencies = GetAllDependenciesForObject(objectAddress);
```

Comment:
We can update the comment of the function. We don't include only objects of pg_dist_object anymore.